inference-engine/ie_bridges/python/src/openvino/offline_transformations/offline_transformations_api_impl.cpp

```diff
@@ -33,6 +33,14 @@ void InferenceEnginePython::ApplyLowLatencyTransformation(InferenceEnginePython:
     manager.run_passes(network.actual->getFunction());
 }
 
+void InferenceEnginePython::ApplyMakeStatefulTransformation(InferenceEnginePython::IENetwork network,
+                                                            std::map<std::string, std::string>& param_res_names) {
+    ngraph::pass::Manager manager;
+    manager.register_pass<ov::pass::MakeStateful>(
+        ov::pass::MakeStateful::find_param_results_by_names(network.actual->getFunction(), param_res_names));
```

Comment:
Why this pass can't just handle this map internally?

Reply:
done